request_a_govuk_domain/settings.py

```diff
@@ -35,7 +35,13 @@
 SECRET_KEY: str = str(uuid.uuid4()) if DEBUG else env.str("SECRET_KEY")
 
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=["*"])
-ENVIRONMENT = env.str("ENVIRONMENT", default=None)
+ENVIRONMENT = env.str("ENVIRONMENT", default=("dev" if DEBUG else None))
+
+# AWS related settings
+IS_AWS: bool = env.bool("IS_AWS", default=False)
+AWS_STORAGE_BUCKET_NAME = env.str(
+    "AWS_STORAGE_BUCKET_NAME", default=f"registration-app-media-root-{ENVIRONMENT}"
```

Comment:
Should we not be getting this from env `S3_MEDIA_ROOT` - thats set in the fargate task container